cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,16 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms")
```

Comment:
We should follow some basic naming conventions when making variable names... This isn't mandatory, but instead of `EXEC`, we should do something like `LINUX_SHORTCUT_EXEC`, `LINUX_SHORTCUT_ICON`, `LINUX_SHORTCUT_ICON_XPM`.  Also, what is `lmms`?  Is that some automagic icon?  Do we not need to link to the SVG/PNG?  Is the 32_32 a backwards compat thing?  Or is this a 32x32 pixel icon?  Is is always an XPM?